### PR TITLE
cache kryo input

### PR DESCRIPTION
**Pull request by chaokunyang**
Cache kryo input as suggested in https://github.com/chaokunyang/fury-benchmarks/commit/1b2b07226b333fa66e3218c502c9416be91769b2#r138421967

Benchmark result:
```
Benchmark                                   Mode  Cnt         Score         Error  Units
KryoBenchmark.furyDeserializeMediaContent  thrpt    9   2468765.648 ±  314266.100  ops/s
KryoBenchmark.furyDeserializeStruct        thrpt    9  20031536.933 ± 1893069.982  ops/s
KryoBenchmark.furySerializeMediaContent    thrpt    9   4041446.580 ±  402177.778  ops/s
KryoBenchmark.furySerializeStruct          thrpt    9  25475238.500 ±  920132.134  ops/s
KryoBenchmark.kryoDeserializeMediaContent  thrpt    9    545891.524 ±   19581.938  ops/s
KryoBenchmark.kryoDeserializeStruct        thrpt    9   2496417.446 ±   76754.451  ops/s
KryoBenchmark.kryoSerializeMediaContent    thrpt    9    478720.501 ±   69891.951  ops/s
KryoBenchmark.kryoSerializeStruct          thrpt    9   1942266.459 ±   63863.902  ops/s
```
bench_struct_tps
bench_media_content_tps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,14 +54,15 @@ microstream structBytes size 120
...
-MicrostreamBenchmark.microstreamSerializeStruct          thrpt    3    233374.717 ±   732378.317  ops/s
+Benchmark                                   Mode  Cnt         Score         Error  Units
+KryoBenchmark.furyDeserializeMediaContent  thrpt    9   2468765.648 ±  314266.100  ops/s
+KryoBenchmark.furyDeserializeStruct        thrpt    9  20031536.933 ± 1893069.982  ops/s
+KryoBenchmark.furySerializeMediaContent    thrpt    9   4041446.580 ±  402177.778  ops/s
+KryoBenchmark.furySerializeStruct          thrpt    9  25475238.500 ±  920132.134  ops/s
+KryoBenchmark.kryoDeserializeMediaContent  thrpt    9    545891.524 ±   19581.938  ops/s
+KryoBenchmark.kryoDeserializeStruct        thrpt    9   2496417.446 ±   76754.451  ops/s
+KryoBenchmark.kryoSerializeMediaContent    thrpt    9    478720.501 ±   69891.951  ops/s
+KryoBenchmark.kryoSerializeStruct          thrpt    9   1942266.459 ±   63863.902  ops/s
```
Comment (NathanSweet) [inline README.md:65]: The microstream section now has Kryo results.

Comment (NathanSweet) [reply README.md:65]: Any chance the charts can include +/- error bars? ([like this](https://raw.github.com/wiki/EsotericSoftware/kryo/images/benchmarks/string.png))

Comment (chaokunyang) [reply README.md:65]: > like this

My bad, it's midnight in my timezone, I updated the wrong place.

Comment (chaokunyang) [reply README.md:65]: > Any chance the charts can include +/- error bars? ([like this](https://raw.github.com/wiki/EsotericSoftware/kryo/images/benchmarks/string.png))

[The errorbar](https://raw.github.com/wiki/EsotericSoftware/kryo/images/benchmarks/string.png) is be great, I will take a look at how to implement it using matplotlib

**Pull request merged**